sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -118,12 +120,27 @@ export function useFormatters() {
     };
   };
 
+  /**
+   * Convert a string or date (Z  OR 00:00 or 0 offset or date only)
+   * to a string format like "Aug 05 2021".
+   * @param date string or date to be converted.
+   * @returns string representation (e.g. Aug 05 2021).
+   */
+  const originalDateOnlyLongString = (date: string | Date): string => {
+    if (date) {
+      return dayjs(date)
+        .utc()

```

Comment:
the only difference this method and the dateOnlyLongString has is the addition of the offset, can we have the old method dateOnlyLongString add the offset value ? so that the dates will all be synchronized in the front end.

Reply:
this function is removed